opus/application/test_api/responses/results_nh_lorri_lor_0299075349_files.json

```diff
@@ -27,6 +27,13 @@
                 "https://opus.pds-rings.seti.org/holdings/previews/NHxxLO_xxxx/NHPELO_2001/data/20150713_029907/lor_0299075349_0x630_sci_med.jpg",
                 "https://opus.pds-rings.seti.org/holdings/previews/NHxxLO_xxxx/NHPELO_2001/data/20150713_029907/lor_0299075349_0x630_sci_full.jpg"
             ],
+            "nh_lorri_documentation": [
+                "https://opus.pds-rings.seti.org/holdings/documents/NHxxLO_xxxx/Weaver-etal-2008-SSR.pdf",
+                "https://opus.pds-rings.seti.org/holdings/documents/NHxxLO_xxxx/Morgan-etal-2005-SPIE.pdf",
+                "https://opus.pds-rings.seti.org/holdings/documents/NHxxLO_xxxx/LORRI-True-Exposure-Times.pdf",
```

Comment:
There are several .png files in the NHLORRI documents directory that should be included here.